HACKING.pod

```diff
@@ -22,13 +22,39 @@ here. However, all new code should be made to comply.
 
 =head1 GETTING STARTED
 
-In the B<dev_scripts> directory you will find scripts that are meant to be
-useful for developing. Most of them are tools for creating and using
-Munin in a sandbox.
+
+=head2 GITHUB
+
+The main development of Munin is happening on L<https://github.com>. With a
+free account you can report feature requests and issues
```

Comment:
Nitpicking: I would not like to the see the word `free` attached to a proprietary service.

How about "After registering an account"?